test/integration/agent-integration.test.ts

```diff
@@ -112,10 +113,12 @@ describe('Integration test', () => {
         await page.click('#testRadioButton')
 
         const domSnapshot = await snapshot(page, 'Serialize input elements')
-        expect(domSnapshot).to.contain('test input value')
-        expect(domSnapshot).to.contain('type="checkbox" checked')
-        expect(domSnapshot).to.contain('type="radio" checked')
-        expect(domSnapshot).to.contain('test textarea value')
+        const $ = cheerio.load(domSnapshot)
```

Comment:
[Cheerio](https://github.com/cheeriojs/cheerio) makes working with the _string_ of the DOM much easier by giving a jQuery API to transverse the string. This was needed because the `contain` method checks to make sure that _exact_ string matches. So the order of `type="radio" checked` needs to be exact (which it isn't) to pass.

Cheerio allows us to query for that element in the DOM and check its attributes.